requests/caps.js

```diff
@@ -101,9 +101,27 @@ export const getAbandonoMensal = async (municipioIdSus) => {
   }
 };
 
-export const getAbandonoCoortes = async (municipioIdSus) => {
+export const getPerfilAbandono = async (municipioIdSus) => {
+  try {
+    const endpoint = "/abandono/resumo?municipio_id_sus=" + municipioIdSus;
+
+    const { data } = await axiosInstance.get(endpoint);
+
+    return data;
+  } catch (error) {
+    console.log('error', error.response.data);
+  }
+};
+
+export const getAbandonoCoortes = async ({
+  municipioIdSus,
+  periodo,
+  estabelecimento
+}) => {
   try {
-    const endpoint = "/abandono/coortes?municipio_id_sus=" + municipioIdSus;
+    let endpoint = '/abandono/coortes?municipio_id_sus=' + municipioIdSus;
+    endpoint = addQueryParamSeExiste(endpoint, 'periodo', periodo);
+    endpoint = addQueryParamSeExiste(endpoint, 'estabelecimento', estabelecimento);
```

Comment:
@jvns2010, executando o código localmente, notei que que os dados entregues pela API não eram apenas os de `periodo = Último período` e `estabelecimento = Todos`.

Percebi que os nomes dos parâmetros opcionais do endpoint refatorado `/abandono/coortes` são `periodos` e `estabelecimentos`, como determinamos para padronização, mas, ao checar os parâmetros existentes para adicioná-los na requisição, foram usados os nomes `periodo` e `estabelecimento`.

Com isso, sugiro a seguinte alteração para a requisição feita na página de abandono puxe a menor quantidade possível de dados da API:

```suggestion
    endpoint = addQueryParamSeExiste(endpoint, 'periodos', periodo);
    endpoint = addQueryParamSeExiste(endpoint, 'estabelecimentos', estabelecimento);
```